tests/org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/formatting/TypeReferenceFormatterTest.xtend

```diff
@@ -21,7 +21,7 @@ class TypeReferenceFormatterTest {
 				{
 					var �toBeFormatted� x
 				}
-			'''
+			'''.toString.trim
```

Comment:
I think it's ok to delete all WS after the last token but preserve comments.
Question: how does it behave if the expression is embedded into another DSL, but special formatting is specified?